### PR TITLE
Add newsletter and submit button

### PR DESCRIPTION
**Pull request by F-Kirchhoff**

### CODE REVIEW

**Comment by F-Kirchhoff** [comment]
also fixed wrong label connections

File: index.html [hunk 1]
```diff
@@ -52,77 +65,108 @@ <h1 id="form-header">
           <label for="interests">Ich interessiere mich fuer ...*</label>
           <ul id="interests">
             <li>
-              <input type="checkbox" id="webdevopment" />
+              <input type="checkbox" id="webdevopment" name="webdevelopment" />
```
Comment (derjubu) [inline index.html]: The value="" attribute for the checkboxes in the interest section ist missing.

File: index.html [hunk 2]
```diff
@@ -52,77 +65,108 @@ <h1 id="form-header">
...
+              />
+              <label for="data-analytics">Data Analytics</label>
             </li>
             <li>
-              <input type="checkbox" id="cloud-development" />
-              <label for="webdevopment">Web Development</label>
+              <input
+                type="checkbox"
+                id="cloud-development"
+                name="cloud-developmWeb Developmentent"
```
Comment (derjubu) [inline index.html]: Typo; should be name="cloud-development"

**Review by derjubu — APPROVED** [review]
Instead of using lists, you can also group the sections with the fieldset element (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset).

File: index.html [hunk 3]
```diff
@@ -17,7 +17,13 @@ <h1 id="form-header">
           <label for="anrede-container">Anrede*</label>
           <ul id="anrede-container">
             <li>
-              <input type="radio" name="anrede" id="anrede-m" value="male" />
+              <input
+                required
+                type="radio"
+                name="anrede"
```
Comment (badabam) [inline index.html]: Is typically in english as it is code and not user facing

File: index.html [hunk 4]
```diff
@@ -17,7 +17,13 @@ <h1 id="form-header">
           <label for="anrede-container">Anrede*</label>
```
Comment (badabam) [inline index.html]: Please look up `fieldset` and `legend` to group multiple radio inputs.

File: index.html [hunk 5]
```diff
@@ -30,18 +36,25 @@ <h1 id="form-header">
...
           />
         </li>
         <li>
-          <label for="vorname">Nachname*</label>
-          <input id="nachname" type="text" placeholder="zb. Mustermann" />
+          <label for="nachname">Nachname*</label>
+          <input
+            required
+            id="nachname"
+            type="text"
```
Comment (badabam) [inline index.html]: `type="text"` is default and could be ommited. But it's not wrong to leave it here.

File: index.html [hunk 6]
```diff
@@ -17,7 +17,13 @@ <h1 id="form-header">
           <label for="anrede-container">Anrede*</label>
           <ul id="anrede-container">
```
Comment (badabam) [inline index.html]: In this case I wouldn't go with lists and list items. Most people won't, I guess.

File: index.html (hunk 1 above)
Comment (badabam) [reply index.html]: It's okay to leave out values for checkboxes, if they have different names.

**Review by badabam — CHANGES_REQUESTED** [review]
Nice work! I left some comments for small improvements.

**Pull request merged**